package.json

```diff
@@ -98,7 +98,7 @@
     "@types/split2": "^2.1.6",
     "@types/stream-buffers": "^3.0.3",
     "@types/tmp": "^0.1.0",
-    "@types/ws": "^6.0.1",
+    "@types/ws": "^7.2.2",
```

Comment:
You'll also need to `npm install` to update the `package-lock.json` file

Reply:
Agh! My plan to do this PR entirely inside the GitHub web UI is foiled... 😄

OK, I've now done it the traditional way and updated the PR.